### PR TITLE
add smartnews logo

### PR DESCRIPTION
**Pull request by ianwith**

### CODE REVIEW

**Comment by swyxio** [comment]
Deploy Preview is done!

Built with commit 13b2ce18e399d2ff49f04533934ed3fe174bbeec

https://deploy-preview-410--svelte-community.netlify.app

File: whos-using-svelte/organisations/smartnews.svg [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
...HOfs7TlxDa918Sg3y6umUde+ZOef18yZr5hqFIgCUeCYAtXsrczemzOqd/mam8XUQJ214+RuwW7uas7PpTl1Xq17LK7BnEu0H9MXfGEFYgpe+OUtHr17xzU3i6lV1PY1d3Y828NrzjmDchoeO6+1Lh7llNeo612yvz9X+37P3M/cc+393dMRBaLAMytQjd2ZZ11de9bX1WpuFo9qnu84OVC6wO+Fsz28Jq7BOS7R77jroTf4JgrkD/ybvMgD1+je+UrOe+CgtoeD98z52jNeayuxejT8HpfM5XOUP9rj/TO+st9sfmpRIApEgZEC1QiO+lbyK2uNelbztW8We22Pd/Wjua5fupEHPYeuXjuaoz/4hgrEALzhS73iSt33YCXnPffmutbKHrUPOXzuqGeW36uxD1j3I/8I/Mu9H3G/7BEFosBvBTqj97vrPpkje896R7War7FuVXMej3idN+o7K1/3U6zh618yfY5a8EMUyB/zD3nRV1yz+25cm/N5znWsWTyrdXNvzWm+Rt33kv39udr3e+Ylc+v80brJR4EoEAXOUqAzkEfWXp0/6+tq1+bqPI+d646zeFZDn9rTrTnKsUbwgxWISfjgl3/F1bvvS5dj6a5WczXW3LNzozVn56Tm2J3L63v81vl766ceBaJAFLi3Ap3xPLLn6vxZ36jW5c/Mra6FHkf7mReMAl8KxDTki3CGAqPv0SivPUe1s/KzPfzOo/28B36klzmreM+1V8+QvigQBaKAFOjM5VnKHFl7pXfWM6qdlZ9pNdrjLB2zzgcoEGPwAS/5Ca44+p6N8hz51rrW2VuDvY72+rzKj+xZ5yaOAlEgCryiAmeZ0iPrrPTu9Vxb35v3iu8wZ34yBWImnuyFfPBxZt/FWa1KdqSXudfMYe4e3nPtvb1TjwJRIArcU4F7GtVr1j4yZ9Y7q91Tz6wdBX4oEAPxQ44EL6TAynd3pefIlc9e78je6Y0CUSAKvIMCZxvglfVWet5B29zhzRSI6XizF5rrXKXAmf87OHOtqy6TSVEgCkSBP1LgTDN85lp/JEe2jQK3KfD/AesdoPsEYTsaAAAAAElFTkSuQmCC" opacity="1" x="109" y="111" width="745" height="744" preserveAspectRatio="none" clip-path="url(#Clip_8)"/>
```
Comment (dreitzner) [inline whos-using-svelte/organisations/smartnews.svg]: Is it possible to export the svg without png images in them?
Or at least make it smaller than 105kB :)

**Review by dreitzner — CHANGES_REQUESTED** [review]

File: whos-using-svelte/organisations/smartnews.svg (hunk 1 above)
Comment (ianwith) [reply whos-using-svelte/organisations/smartnews.svg]: @dreitzner Thank you!
I have compressed the SVG image to 102K. if it is still too big, I can try PNG format, which is about 12K.

Comment (dreitzner) [reply whos-using-svelte/organisations/smartnews.svg]: 12kB sounds reasonable to me 👍

Comment (ianwith) [reply whos-using-svelte/organisations/smartnews.svg]: @dreitzner changed svg to png.

Comment (ianwith) [reply whos-using-svelte/organisations/smartnews.svg]: by the way, is it possible to make it prominent?

**Review by dreitzner — APPROVED** [review]

**Comment by dreitzner** [comment]
LGTM

**Comment by ianwith** [comment]
@dreitzner are you able to merge this PR? Thank you!

**Pull request merged**